File: src/ServiceControl/Recoverability/Retrying/History/HistoricRetryOperation.cs
namespace ServiceControl.Recoverability
{
    using System;

    public class HistoricRetryOperation
    {
        public string RequestId { get; set; }
        public RetryType RetryType { get; set; }
        public DateTime StartTime { get; set; }
        public DateTime CompletionTime { get; set; }
        public string Originator { get; set; }
        public bool Failed { get; set; }
        public int NumberOfMessagesProcessed { get; set; }
    }
}

File: src/resources/HistoricRetryOperation.ts
import { RetryType } from "./RetryType";

export default interface HistoricRetryOperation {
  request_id: string;
  retry_type: RetryType;
  start_time: string;
  completion_time: string;
  originator: string;
  failed: boolean;
  number_of_messages_processed: number;
}
